spec/values/calculation/atan2.hrx

```diff
@@ -0,0 +1,200 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `atan2()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> units/none/input.scss
+a {b: atan2(1, -10)}
+
+<===> units/none/output.css
+a {
+  b: 174.2894068625deg;
+}
+
+<===>
+================================================================================
+<===> units/compatible/input.scss
+a {b: atan2(1cm, -10mm)}
+
+<===> units/compatible/output.css
+a {
+  b: 135deg;
+}
+
+<===>
+================================================================================
+<===> units/real_and_unknown/input.scss
+a {b: atan2(1px, 10%)}
+
+<===> units/real_and_unknown/output.css
+a {
+  b: atan2(1px, 10%);
+}
+
+<===>
+================================================================================
+<===> units/unknown/input.scss
+a {
+  b: atan2(1%, 2%);
+}
+
+<===> units/unknown/output.css
+a {
+  b: atan2(1%, 2%);
+}
+
+<===>
+================================================================================
+<===> units/fake/input.scss
```

Comment:
I added this test case for all two-argument functions. In the case of `atan2`, it returns the result of the calculation with the `deg` unit, and the other functions return the result with the `foo` unit. This is happening because we are only checking if the units are _compatible_, so two numbers with the `foo` unit would be compatible.
I do see that the spec specifies `round`, `mod`, and `rem` accept any units that are mutually compatible. However, `atan2` and `hypot` do require known units. Is there a function in to identify real units?

Reply:
I believe the existing behavior is correct. "Known units" is [defined as](https://github.com/sass/sass/blob/main/accepted/calc-functions.md#known-units) "A number has _known units_ unless it has unit `%`.", so `foo` counts.